Task2/aga/TicTacToe/TicTacToe/Click.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicTacToe
+{
+    internal class Click
+    {
+        private int turn = 1;
+        public Click()
+            { }
+
+        public void TurnCounter()
```

Comment:
As per above comment, this method does not have verb in its name, so the reader is not sure what it is doing exactly.